lib/ui/src/components/sidebar/SidebarItem.js

```diff
@@ -111,9 +111,19 @@ export default function SidebarItem({
   }
 
   return (
-    <Item isSelected={isSelected} depth={depth} loading={loading} {...props}>
-      <Expander isExpandable={!isLeaf} isExpanded={isExpanded ? true : undefined} />
-      <Icon icon={iconName} isSelected={isSelected} />
+    <Item
+      isSelected={isSelected}
+      depth={depth}
+      loading={loading}
+      {...props}
+      className={isSelected ? 'sidebar-item selected' : 'sidebar-item'}
```

Comment:
I'm ok, with this, though I think it'd be safer to encapsulate the adding of these classNames to the components. That way we can just use them, and we don't have to worry about adding these classnames on the jsx.

WDYT?

Reply:
I tried on adding classNames to components when i started working on this change, but i overcame some issues of initial styles not getting applied. So, i went ahead and applied classNames this way, which was not breaking in any of the cases.